src/resources/views/crud/columns/range.blade.php

```diff
@@ -0,0 +1,22 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['text'] = $column['default'] ?? '-';
+    $max_value = (isset($column['max'])) ? $column['max'] : '100';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
+        $column['value'] = json_encode($column['value']);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = ($column['value'] / $max_value) * 100;
+    }
+@endphp
+
+@if ($column['text'] != "-")
```

Comment:
And when column is empty ( `-` ) what do we show ?

Reply:
Should I add to show "-" default in this case?

Reply:
I think it's better for consistency with the other columns.

Reply:
Added

Reply:
Applied as suggested